src/corona_hakab_model/generation/circles_consts.py

```diff
@@ -74,33 +109,32 @@ def from_file(cls, param_path):
         return cls(**parameters)
 
     def get_geographic_circles(self):
-        assert self.geo_circles_amount == len(self.geo_circles_names)
+        assert self.geo_circles_amount == len(self.geo_circles)
         return [
             GeographicalCircleDataHolder(
-                self.geo_circles_names[i],
+                geo_circle["name"],
                 self.geo_circles_agents_share[i],
-                self.get_age_distribution(),
-                self.circle_size_distribution_by_connection_type,
-                self.get_connection_types_prob_by_age(),
+                self.get_age_distribution(geo_circle),
+                geo_circle["circle_size_distribution_by_connection_type"],
+                self.get_connection_types_prob_by_age(geo_circle),
                 self.multi_zone_connection_type_to_geo_circle_probability[i],
-                self.get_required_adult_distributions(),
-                self.teachers_workforce_ratio
+                self.get_required_adult_distributions(geo_circle),
+                geo_circle["teachers_workforce_ratio"]
             )
-            for i in range(self.geo_circles_amount)
+            for i, geo_circle in enumerate(self.geo_circles)
         ]
 
-    def get_age_distribution(self):
-        return rv_discrete(10, 70, values=(self.ages, self.age_prob))
+    def get_age_distribution(self, geo_circle):
+        return rv_discrete(10, 70, values=(geo_circle["ages"], geo_circle["age_prob"]))
```

Comment:
10, 70 could be switched with: min /max (geo_circle["ages"]).

Reply:
@tal-nc  Why have those values at all if you specifies values?
@RoyIsland

Reply:
good point. it isn't necessary.
tbh we don't even use this range anywhere in the code (as far as i know).